microprofile/WFLY-12758_integrate_opentracing.adoc

```diff
@@ -0,0 +1,111 @@
+= Integrate Microprofile OpenTracing 1.3
+:author:            Emmanuel Hugonnet
+:email:             ehugonne@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          microprofile,opentracing,observability,tracing,openshift
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Currently the integration of Microprofile Opentracing is minimal as you can't configure the tracer using the WildFly Management Model.
+So we will implement a way to define several tracer configurations (Jaeger) to be used in deployments.
+One of those configuration will become a default one so that deployments that don't define which configuration to use will use this one.
```

Comment:
What happens when deployments defines configuration to use but that configuration is not present on the server ?

not present can mean
 - configuration not yet present
 - configuration not present anymore (new one was introduced)
 - typo in configuration name - web.xml or server config

Reply:
Then it fails since we create a MSC service dependency on that configuration. If none is defined we fall back to the default one if one is defined otherwise no configuration is provided